lib/detect-dependencies.js

```diff
@@ -175,8 +175,10 @@ function gatherInfo(config) {
 
     if (dep.main.length === 0 && !depIsExcluded) {
       // can't find the main file. this config file is useless!
-      config.get('on-main-not-found')(component);
-      return;
+      onfig.get('on-main-not-found')(component);
+      // Polymer/paper-elements comes with a missing main but has useful dependencies
+      // skipping the return correctly includes the transient dependencies
+      // return;
```

Comment:
bower.json: We've been a little inconsistent here. I'm talking to the team
about getting our house in order. Sorry about that!
On 28 Jun 2015 20:20, "Stephen Sawchuk" <notifications@github.com> wrote:

> In lib/detect-dependencies.js
> <https://github.com/taptapship/wiredep/pull/198#discussion_r33425911>:
>
> > @@ -175,8 +175,10 @@ function gatherInfo(config) {
> >
> >      if (dep.main.length === 0 && !depIsExcluded) {
> >        // can't find the main file. this config file is useless!
> > -      config.get('on-main-not-found')(component);
> > -      return;
> > +      onfig.get('on-main-not-found')(component);
> > +      // Polymer/paper-elements comes with a missing main but has useful dependencies
> > +      // skipping the return correctly includes the transient dependencies
> > +      // return;
>
> You can just remove all the comments.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/taptapship/wiredep/pull/198/files#r33425911>.
>